packages/ui/src/DropdownMenu/checkbox-item.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  CheckboxItem as RadixDropdownMenuCheckboxItem,
+  ItemIndicator as RadixDropdownMenuItemIndicator,
+} from '@radix-ui/react-dropdown-menu';
+import { ReactNode } from 'react';
+import { Check } from 'lucide-react';
+import { asTransientProps } from '../utils/asTransientProps.ts';
+import { Text } from '../Text';
+import { DropdownMenuItemBase, MenuItem } from './shared.ts';
+
+export interface DropdownMenuCheckboxItemProps extends DropdownMenuItemBase {
+  children?: ReactNode;
+  checked?: boolean;
+  onChange?: (value: boolean) => void;
+}
+
+export const CheckboxItem = ({
```

Comment:
suggestion: keep the dropdown menu open when selecting checkbox items

I couldn't figure out a way to do this in the docs, nor do I see in the code where you're closing the dropdown menu once a user clicks a menu item (maybe Radix does it automatically?). But it'd be nice to keep the dropdown menu open when clicking checkbox items specifically, since the user may wish to click more than one.

Not a blocker for this PR, though !

Reply:
We should probably investigate the use cases with time. Leaving as-is for now